src/steamship/plugin/outputs/training_parameter_plugin_output.py

```diff
@@ -17,6 +18,17 @@ class TrainingParameterPluginOutput(CamelModel):
     inference_params: Dict[str, Any] = None
     export_request: ExportPluginInput = None
 
+    @staticmethod
+    def from_input(input: TrainingParameterPluginInput) -> TrainingParameterPluginOutput:
```

Comment:
This can also be done using `TrainingParameterPluginOutput.parse_obj(input.__dict__)`. The verbosity helps in this case so I leave it up to you to decide what feels best.

Reply:
Thanks for the pointer!  I think I'll leave it in this case, because there may be specific transformations here.